azavea/climate-change-lab
# Update climate-change-components to the most recent version

## Overview

Upgrades to the latest version of the `climate-change-components`.

Also switches over to making indicator API calls using lat/lon queries instead of using the City ID to query.

For now it is still required to have an imported city to view data, but this should make it easier to transition away from that requirement in #333 .


### Demo

Checkout the lat/lon in the URL here!

![image](https://user-images.githubusercontent.com/4432106/52881683-3ee81100-3133-11e9-9b59-9f5d59cfaf06.png)

Navbar now wraps at smaller screen sizes
![localhost_4200_](https://user-images.githubusercontent.com/4432106/52883177-0f87d300-3138-11e9-90db-01b23448bbac.png)

New project form is wider and the column sizes changed a bit
![localhost_4200_ 2](https://user-images.githubusercontent.com/4432106/52883205-20d0df80-3138-11e9-8189-f237aeef6209.png)


### Notes

 - Upgrading the components also required upgrading `ngx-bootstrap` to `2.x`. I read over the release notes and didn't note any breaking changes that would apply to us.
 - The longer labels on the scenario toggle require some small CSS tweaks, which I consulted with @alexelash about


## Testing Instructions

 * `yarn`
 * `yarn run serve`
 * You should still be able to view data for every indicator page
 * You should still be able to create a new project

## Checklist

Closes #332
Closes https://github.com/azavea/climate-change-lab/issues/316
